content/en/docs/collector/management.md

```diff
@@ -81,129 +81,207 @@ Let's have a look at a concrete setup:
    the server-side OpAMP part and the collector (or a supervisor controlling the
    collector) implementing OpAMP client-side.
 
+### Quick start
+
 You can try out a simple OpAMP setup yourself by using the [OpAMP protocol
 implementation in Go][opamp-go]. For the following walkthrough you will need to
-have Go in version 1.19 or above available.
+have Go in version 1.22 or above available.
```

Comment:
Version taken from `opamp-go`'s [contribution guidelines](https://github.com/open-telemetry/opamp-go/blob/d646431dacb432f56a84111bf94c9741692f53c0/CONTRIBUTING.md?plain=1#L11) and [go.mod](https://github.com/open-telemetry/opamp-go/blob/d646431dacb432f56a84111bf94c9741692f53c0/go.mod#L3).

Reply:
```suggestion
Go 1.22+.
```